samples/node/square/README.adoc

```diff
@@ -6,32 +6,26 @@
 *Protocol*:: HTTP
 *Input*:: numbers
 
-=== Build the Docker container for the function code
+=== prerequisites
 
-```
-./riff build -n square -v 0.0.2 -f samples/node/square
-```
-
-=== Create the Function and Topic(s)
-
-```
-./riff apply -f samples/node/square
-```
+1. riff is deployed
+2. the `riff` script is on your PATH
+3. the working directory is `samples/node/square`
 
-=== Publish a Message
+=== create the function and its input topic
 
 ```
-./request numbers 2
+riff create
 ```
 
-=== Tail the Sidecar Log
+=== publish a message and wait for the result
 
 ```
-./riff logs -n square -t
+riff publish -d 99 -t
 ```
 
-=== Delete the function
+=== delete the function and its input topic
 
 ```
-./riff delete -n square
+riff delete -n square --all
```

Comment:
you only need `riff delete --all` since the `--all` option uses the $FNPATH which  defaults to current dir unless `-f` is specified

Reply:
The usage was updated in the help text

```
Usage:

  riff delete -n <name>
    or
  riff delete -f <path> --all
```

Reply:
gotcha